_positions/Innovation-Specialist-GS-13.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Innovation Specialist, GS-0301-13
+permalink: who-we-are-hiring/positions/Innovation-Specialist-GS-13/
+team: 18F
```

Comment:
The [content designer example](https://github.com/18F/joining-18f/edit/18f-pages/_positions/content-designer.md) has the value `design` instead of `18F`. Does it still make sense to talk about teams? If so, how can we do so consistently?